src/main/resources/db/migration/V2024_03_18_1456__CIV-13111_defendant_response_full_part_admit_installments_org_com_claimant_scenario.sql

```diff
@@ -0,0 +1,33 @@
+/**
+ * Add scenario
+ */
+INSERT INTO dbs.scenario (name, notifications_to_delete, notifications_to_create)
+VALUES ('Scenario.AAA7.DefResponse.OrgOrLtdCompany.FullOrPartAdmit.PayByInstallments.Claimant',
+        '{"Notice.AAA7.DefResponse.ResponseTimeElapsed.Claimant", "Notice.AAA7.ClaimIssue.Response.Await","Notice.AAA7.ClaimIssue.HWF.PhonePayment","Notice.AAA7.DefResponse.MoretimeRequested.Claimant","Notice.AAA7.ClaimIssue.HWF.FullRemission"}',
+        '{"Notice.AAA7.DefResponse.OrgOrLtdCompany.FullOrPartAdmit.PayByInstallments.Claimant": ["defendantAdmittedAmount", "installmentAmount", "paymentFrequency","firstRepaymentDate"]}');
+
+/**
+ * Add notification template
+ */
+INSERT INTO dbs.dashboard_notifications_templates (template_name, title_En, title_Cy, description_En, description_Cy
+                                                  ,notification_role)
+VALUES ('Notice.AAA7.DefResponse.OrgOrLtdCompany.FullOrPartAdmit.PayByInstallments.Claimant',
+        'Response to the claim', 'Response to the claim',
+        '<p class="govuk-body">The defendant has offered to pay ${defendantAdmittedAmount} in instalments of ${installmentAmount} every ${paymentFrequency}.They are offering to do this starting from ${firstRepaymentDate}.</p><p class="govuk-body">The defendant needs to send you their financial details. <a href="{VIEW_AND_RESPOND}" rel="noopener noreferrer" class="govuk-link">View and respond</a></p>',
+        '<p class="govuk-body">The defendant has offered to pay ${defendantAdmittedAmount} in instalments of ${installmentAmount} every ${paymentFrequency}.They are offering to do this starting from ${firstRepaymentDate}.</p><p class="govuk-body">The defendant needs to send you their financial details. <a href="{VIEW_AND_RESPOND}" rel="noopener noreferrer" class="govuk-link">View and respond</a></p>',
+        'CLAIMANT');
+
+INSERT INTO dbs.task_item_template (task_name_en, category_en, task_name_cy, category_cy, template_name,
+                                    scenario_name, task_status_sequence, role, task_order)
+values ('<a href={VIEW_CLAIM_URL} rel="noopener noreferrer" class="govuk-link">View the claim</a>', 'The claim','<a href={VIEW_CLAIM_URL}>View the claim</a>',
+        'The claim', 'Claim.View', 'Scenario.AAA7.DefResponse.FullAdmit.PayImmediately.Claimant', '{3, 3}', 'CLAIMANT', 1),
+       ('<a href={VIEW_INFO_ABOUT_CLAIMANT} rel="noopener noreferrer" class="govuk-link">View information about the claimant</a>', 'The claim','<a href={VIEW_INFO_ABOUT_CLAIMANT_URL}>View information about the claimant</a>',
+        'The claim', 'Claim.Claimant.Info', 'Scenario.AAA7.DefResponse.FullAdmit.PayImmediately.Claimant', '{3, 3}', 'CLAIMANT', 2),
+       ('<a href={VIEW_RESPONSE_TO_CLAIM} rel="noopener noreferrer" class="govuk-link">View the response to the claim</a>', 'The response','<a href={VIEW_RESPONSE_TO_CLAIM}>View the response to the claim</a>',
```

Comment:
@ElifDenizEsea I think this it the only one you need to add the task item with (VIEW_RESPONSE_TO_CLAIM), I don't think you need the other task_item_templates

Reply:
done